less/core/article.less

```diff
@@ -1,11 +1,16 @@
 .article {
-  &.has-bg-image {
-    background-repeat: no-repeat;
-    background-size: cover;
-    background-position: center top;
+  position: relative;
+
+  &.has-bg-image > .background {
```

Comment:
What's the thinking here?

Reply:
There's a default style for use which covers the biggest majority of use cases. If a use case falls outside of the default, it can be overridden inline in the json using `_backgroundStyles`

Reply:
Yeah, I get that that configuration is the most common for us but generally speaking we don't know thats the case. Depends how you thunk about the theme I guess.

Reply:
i think it's a fairly safe thing to remain in the theme